src/pybind/mgr/dashboard/services/nvmeof_conf.py

```diff
@@ -58,3 +58,21 @@ def remove_gateway(cls, name):
             raise NvmeofGatewayDoesNotExist(name)
         del config['gateways'][name]
         cls._save_config(config)
+    
+    @classmethod
+    def get_client_cert(cls):
+        client_cert = mgr.get_store('nvmeof/mtls_client_cert')
+        if client_cert:
+            return json.dumps(client_cert).encode()
```

Comment:
Why `json.dumps()`? This should be the raw certificate...

Reply:
you are right, it was not needed